src/config.ts

```diff
@@ -66,7 +66,8 @@ const sweepers: SweeperOptions = {
 
 export const clientOptions: ClientOptions = {
 	// Intents dictate what events the client will receive.
-	intents: GatewayIntentBits.Guilds | GatewayIntentBits.GuildMessages | GatewayIntentBits.GuildVoiceStates,
+	intents: GatewayIntentBits.Guilds | GatewayIntentBits.GuildMessages | GatewayIntentBits.GuildVoiceStates | GatewayIntentBits.DirectMessages,
+	partials: ['CHANNEL'], // required to receive DMs
```

Comment:
Prefer enums over hardcoded strings (`Constants` is from discord.js)
```suggestion
	partials: [Constants.PartialTypes.CHANNEL], // required to receive DMs
```